corehq/apps/translations/app_translations.py

```diff
@@ -1108,6 +1031,95 @@ def _partial_upload(rows, details):
     return msgs
 
 
+def _get_condensed_rows(app, sheet):
+    '''
+    Reconfigure the given sheet into objects that are easier to process.
+    The major change is to nest mapping and graph config rows under their
+    "parent" rows, so that there's one row per case proeprty.
+
+    This function also pulls out case detail tab headers and the case list form label,
+    which will be processed separately from the case proeprty rows.
+
+    Returns a three-item tuple: case property rows, tab header rows, case list form label
```

Comment:
could make this a `namedtuple` (is very low overhead and a just bit more explicit)

Reply:
bc4b19e8672